spikeextractors/extractors/shybridextractors/shybridextractors.py

```diff
@@ -0,0 +1,174 @@
+import os
+
+import numpy as np
+
+from spikeextractors import RecordingExtractor
+from spikeextractors import SortingExtractor
+from spikeextractors.extractors.bindatrecordingextractor import BinDatRecordingExtractor
+from spikeextractors.extraction_tools import save_to_probe_file
+
+try:
+    import hybridizer.io as sbio
+    import hybridizer.probes as sbprb
+    HAVE_SBEX = True
+except ImportError:
+    HAVE_SBEX = False
+
+class SHYBRIDRecordingExtractor(RecordingExtractor):
```

Comment:
you can also derive the class directly from `BinDatRecordingExtractor` so you don't need to reimplement the abstract methods

Reply:
good idea!